src/content/docs/en/guides/cms/microcms.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: microCMS & Astro
+description: Use microCMS to manage your Astro project contents
+type: cms
+stub: true
+service: microCMS
+i18nReady: true
+---
+
+[microCMS](https://microcms.io/) is an API-based headless CMS.
+It accelerates your business by greatly reducing development and operation costs through a comfortable dashboard.
```

Comment:
We like to avoid marketing text in our CMS guides. This should tell the user what it is and does. I have quick suggestion here, but feel free to rewrite it yourself.
```suggestion
[microCMS](https://microcms.io/) is an API-based headless CMS, that let's you define content using schemas, and manage it using the dashboard.
```